docs/framework/additional-apis/index.md

```diff
@@ -20,38 +20,38 @@ author: "mairaw"
 ms.author: "mairaw"
 manager: "wpickett"
 ---
-# Additional Class Libraries and APIs
+
+# Additional class libraries and APIs
+
 The .NET Framework is constantly evolving and in order to improve cross-platform development or to introduce new functionality early to our customers, we release new features out of band (OOB). This topic lists the OOB projects that we provide documentation for.  
   
- In addition, some libraries target specific platforms or implementations of the .NET Framework. For example, the
- [CodePagesEncodingProvider Class](CodePagesEncodingProvider.xml) class  makes code page encodings available to UWP apps developed using the .NET Framework. This topic lists these libraries as well.  
+In addition, some libraries target specific platforms or implementations of the .NET Framework. For example, the [CodePagesEncodingProvider Class](https://msdn.microsoft.com/library/system.text.codepagesencodingprovider.aspx) class makes code page encodings available to UWP apps developed using the .NET Framework. This topic lists these libraries as well.  
   
-## OOB projects  
+## OOB projects
   
-|Project|Description|  
-|-------------|-----------------|  
-|[System.Collections.Immutable](System.Collections.Immutable.xml)|Provides collections that are thread safe and guaranteed to never change their contents.|  
-|[System.Net.Http.WinHttpHandler](http://go.microsoft.com/fwlink/p/?LinkId=624833)|Provides a message handler for [HttpClient Class](HttpClient.xml) based on the WinHTTP interface of Windows.|  
-<!-- todo: fix link -->
-|`System.Numerics.Vectors1`|Provides a library of vector types that can take advantage of SIMD hardware-based acceleration.| 
-<!-- restore this when the doc is found 
-|[TPL Dataflow Library](../Topic/TPL%20Dataflow%20Library.md)|The TPL Dataflow Library provides dataflow components to help increase the robustness of concurrency-enabled applications.|  
--->
+| Project | Description |  
+| ------- | ----------- |  
+| [System.Collections.Immutable](https://msdn.microsoft.com/library/system.collections.immutable.aspx) | Provides collections that are thread safe and guaranteed to never change their contents. |
+| [System.Net.Http.WinHttpHandler](http://go.microsoft.com/fwlink/p/?LinkId=624833) | Provides a message handler for <xref:System.Net.Http.HttpClient> based on the WinHTTP interface of Windows. |
+| <xref:System.Numerics> | Provides a library of vector types that can take advantage of SIMD hardware-based acceleration.| 
```

Comment:
The problem here is that we have to link to the .NET Core package rather than a general one. I think that we need to do this as [System.Numerics.Vectors](https://docs.microsoft.com/en-us/dotnet/api/system.numerics?view=netcore-1.1.0).